templates/partials/committee/financial-summary.html

```diff
@@ -5,14 +5,14 @@
     <div class="content__section">
       <div class="section__heading">
         <h2 class="heading__title" id="section-1-heading">
-          Financial summary - {{ cycle|fmt_year_range }}
+          Financial summary: {{ cycle|fmt_year_range }}
```

Comment:
Good call. This is a change that we should make on all of the tabs on candidate, committee and election pages.